tests/unit/log_test.go

```diff
@@ -9,16 +9,58 @@ import (
 	utils "github.com/pokt-network/pocket-core/tests/unit/utils"
 )
 
+func TestTextLogs(t *testing.T) {
+	if err := logs.Log("Unit test for the json log functionality", logs.InfoLevel, logs.TextLogFormatter); err != nil {
+		t.Fatalf(err.Error())
+	}
+}
+
 func TestJSONLogs(t *testing.T) {
 	if err := logs.Log("Unit test for the json log functionality", logs.InfoLevel, logs.JSONLogFormat); err != nil {
 		t.Fatalf(err.Error())
 	}
 }
 
-func TestTextLogs(t *testing.T) {
-	if err := logs.Log("Unit test for the text log functionality", logs.InfoLevel, logs.TextLogFormatter); err != nil {
+func TestJSONLogLevel(t *testing.T) {
+	if err := logs.Log("Unit test for the trace log functionality", logs.TraceLevel, logs.JSONLogFormat); err != nil {
```

Comment:
The text should indicate the log type as well as the level, e.g.: `Log type JSON Log level TRACE`